ui/uihandler.go

```diff
@@ -513,16 +501,24 @@ func Serve(port int, debugpath string, uipath string) {
 		debugPath = ".." + debugpath // TODO: calculate actual path if not same number of directories
 		http.HandleFunc(uiPath, Handler)
 		fmt.Printf("UI starting - visit:\nhttp://localhost:%d%s\n", port, uiPath)
-		logoPath = uiPath + logoURI
-		http.HandleFunc(logoPath, LogoHandler)
-		chartJSPath = uiPath + chartjsURI
-		http.HandleFunc(chartJSPath, ChartJSHandler)
-		// Use relative paths after that (in the html template):
-		logoPath = "./" + logoURI
-		chartJSPath = "./" + chartjsURI
+
 		fetchPath = uiPath + fetchURI
 		http.HandleFunc(fetchPath, FetcherHandler)
 		fhttp.CheckConnectionClosedHeader = true // needed for proxy to avoid errors
 	}
+	logoPath = "../static/img/logo.svg"
```

Comment:
`./static` instead of `../static` (as a side note I think `static/img/logo.svg` would also work as that's relative too, I forgot why I've put ./ in the current version)

Reply:
Statics are on "/static/js/...". The fortio page is on "/fortio" so we should use "../"

Did you mean putting statics under "/fortio/static/..."?

Reply:
yeah I meant everything related to the ui under the uiPath (/fortio by default) prefix
but I can change it in subsequent PR too